daml-lf/data/src/test/scala/com/digitalasset/daml/lf/data/TryOpsTest.scala

```diff
@@ -1,87 +0,0 @@
-// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
if we leave it around we should imho also leave the tests around.

Reply:
:+1: